uhu/repl/helpers.py

```diff
@@ -27,29 +26,27 @@
     PackageUIDValidator)
 
 
-manager = KeyBindingManager.for_prompt()  # pylint: disable=invalid-name
+registry = KeyBindings()  # pylint: disable=invalid-name
```

Comment:
Is the lint still required?

Reply:
Yes, lint still need, because if remove the pylint happens an error on the registry variable